com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/RestartFrameHandler.java

```diff
@@ -62,11 +63,14 @@
                 stepInto(context, reference);
             } catch (DebugException de) {
                 context.getProtocolServer().sendEvent(new Events.UserNotificationEvent(NotificationType.ERROR, de.getMessage()));
+                throw AdapterUtils.createCompletionException(
```

Comment:
@yaohaizh , please confirm that it is acceptable because these lines changes the original logic.

Reply:
Okay for me here